src/hooks/verify-token.js

```diff
@@ -0,0 +1,34 @@
+import jwt from 'jsonwebtoken';
+import errors from 'feathers-errors';
+
+/**
+ * Verifies that a JWT token is valid
+ * 
+ * @param  {Object} options - An options object
+ * @param {String} options.secret - The JWT secret
+ */
+export default function(options = {}){
+  const secret = options.secret;
+
+  return function(hook) {
+    const token = hook.params.token;
+
+    if (!token) {
+      return Promise.resolve(hook);
```

Comment:
true. I still have tests to write for these hooks so I'll do that at the same time.